src/js/tabs/motors.js

```diff
@@ -472,42 +528,187 @@ TABS.motors.initialize = function (callback) {
             neutral3d = (FC.MOTOR_3D_CONFIG.neutral > 1575 || FC.MOTOR_3D_CONFIG.neutral < 1425) ? 1500 : FC.MOTOR_3D_CONFIG.neutral;
         }
 
-        const motorsWrapper = $('.motors .bar-wrapper'),
-            servos_wrapper = $('.servos .bar-wrapper');
+        const motorsWrapper = $('.motors .bar-wrapper');
 
         for (let i = 0; i < 8; i++) {
             motorsWrapper.append(`\
-                    <div class="m-block motor-${i}">\
-                    <div class="meter-bar">\
-                    <div class="label"></div>\
-                    <div class="indicator">\
-                    <div class="label">\
-                    <div class="label"></div>\
-                    </div>\
-                    </div>\
-                    </div>\
-                    </div>\
+                <div class="m-block motor-${i}">\
+                <div class="meter-bar">\
+                <div class="label"></div>\
+                <div class="indicator">\
+                <div class="label">\
+                <div class="label"></div>\
+                </div>\
+                </div>\
+                </div>\
+                </div>\
             `);
-
-            servos_wrapper.append('\
-                    <div class="m-block servo-' + (7 - i) + '">\
-                    <div class="meter-bar">\
-                    <div class="label"></div>\
-                    <div class="indicator">\
-                    <div class="label">\
-                    <div class="label"></div>\
-                    </div>\
-                    </div>\
-                    </div>\
-                    </div>\
-            ');
         }
 
         $('div.sliders input').prop('min', rangeMin)
         .prop('max', rangeMax);
         $('div.values li:not(:last)').text(rangeMin);
 
-        // UI hooks
+        const featuresElement = $('.tab-motors .features');
+
+        FC.FEATURE_CONFIG.features.generateElements(featuresElement);
+
+        /*
+        *   ESC protocol
+        */
+
+        const escProtocols = [
+            'PWM',
+            'ONESHOT125',
+            'ONESHOT42',
+            'MULTISHOT',
+        ];
+
+        escProtocols.push('BRUSHED');
```

Comment:
Where have the version checks for this gone?

Reply:
Changed (using EscProtocols utils now)